app/preload.js

```diff
@@ -16,15 +16,24 @@ function navigate(url) {
 
 ipcRenderer.on('isPlaying', (event) => {
   const isPlaying = !!document.querySelector('.playing')
-  event.sender.send('isPlaying', isPlaying)
+  const icon = document.getElementsByClassName('playbackSoundBadge__avatar sc-media-image')[0].children[0].children[0].getAttribute('style')
```

Comment:
You're a lot smarter than me my man. I didn't even consider that. Thanks for the heads up